packages/components/stories/ActionIconToggle.js

```diff
@@ -23,12 +23,51 @@ const activeIconToggle = {
 	active: true,
 	icon: 'talend-panel-opener-right',
 	id: 'my-active-action',
-	label: 'Click me, I\'m inactive',
+	label: "Click me, I'm inactive",
 	'data-feature': 'actionicontoggle',
 	onClick: action('You clicked the active button'),
 	tooltipPlacement: 'top',
 };
 
+const ACTION1 = 'Action 1';
+const ACTION2 = 'Action 2';
+
+class DisablingActionIconToggle extends React.Component {
```

Comment:
Why do you add 2 stories to check the same tooltip trigger code ?

Reply:
I test for the ActionButton & for the ActionIconToggle